docs/release-notes.rst

```diff
@@ -10,6 +10,66 @@
  Version 0.19
 **************
 
+Version 0.19.11
+===============
+
+**Release Date:** February 17, 2023
+
+**Bug Fixes**
+
+-  Kubernetes: Fix an issue where environment variables with an equal character in the value such as
+   ``func=f(x)=x`` were processed incorrectly in Kubernetes.
+
+-  Agent: Fix a bug where if the flag ``agent_reattach_enabled`` was enabled and master was down
+   while an active task's docker container failed, the task could get stuck in an unkillable running
+   state.
+
+-  ``det deploy aws``: updated cloudformation permissions to allow checkpoint downloads through
+   master.
+
+-  Tasks: Fix a bug where in rare cases tasks could take an extra 30 seconds to complete.
+
+**Improvements**
+
+-  Container Images: Publish multi-arch master and agent container image manifests with amd64,
+   arm64, and ppc64 architectures.
+
+-  Experiments: If an experiment with no checkpoints is deleted, a checkpoint GC task will no longer
+   be launched. Launching a checkpoint GC task could prevent experiments with certain incorrect
+   configuration from being deleted.
+
+-  Cluster: Capability added for checkpoint downloads from Google Cloud
+      Storage via a master instance
+
+-  Installation: ``.deb`` and ``.rpm`` Linux packages will now install master and agent binaries
+   into ``/usr/bin/`` instead of ``usr/local/bin/``, to be more in-line with Filesystem Hierarchy
+   Standard.
+
+-  Kubernetes: Empty environment variables can now be specified in Kubernetes while before they
+   would throw an error.
+
+-  Kubernetes: zero-slot tasks on gpu clusters will not request ``nvidia.com/gpu: 0`` resources any
+   more, allowing them to be schedule on cpu-only nodes.
+
+-  Installation: added experimental Homebrew (macOS) package.
+
+-  Scheduler: The scheduler can be configured to find fits for distributed jobs against agents of
+   different sizes.
+
+**New Features**
+
+-  CLI: Add support for a ``--add-tag`` flag to AWS ``det deploy aws up``
+
+   -  A new argument is added to ``det deploy aws up`` that allows users to specify tags that should
+      be added to the underlying CloudFormation stack.
+
+   -  New tags will not replace automatically-added tags such as ``deployment-type`` or
+      ``managed-by``
+
+   -  Any added tags that should persist across updates should be always be included when using
+      ``det deploy aws up`` -- if the argument is missing, any previously added tags would be
+      removed
```

Comment:
```suggestion
   -  New tags will not replace automatically added tags such as ``deployment-type`` or
      ``managed-by``.

   -  Any added tags that should persist across updates should be always be included when using
      ``det deploy aws up`` -- if the argument is missing, any previously added tags would be
      removed.
```

Is there a place in the body of the docs that this can go? It's a bit much to stuff into the release notes. If not, I would also suggest that they not be bullet points.